setup.py

```diff
@@ -40,6 +40,18 @@ def _find_msbuild_tool(tool="msbuild.exe", use_windows_sdk=False):
                 locappdir, vcpy27, r"x64\mt.exe")
             if os.path.exists(mtpath):
                 return mtpath      
+        if sys.version_info[:2] == (3,5):
```

Comment:
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots` is where you'll find the install directories of the Windows dev kits. (In 64-bit processes on 64-bit systems it'll be `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows Kits\Installed Roots` - easiest to try the other one first and then fall back to the longer path.)